config/routes.rb

```diff
@@ -34,7 +34,11 @@ def draw_routes(prefix)
             end
           end
 
-          resources :policies, except: [:new, :edit]
+          resources :policies, except: [:new, :edit] do
+            resources :tailorings, only: [:index, :show], parents: [:policy] do
+              get 'tailoring_file', on: :member
```

Comment:
@RoamingNoMaD I think you can use the `on: :collection` similarly in your PR 🤔 not having a strong opinion about it, but agree with @romanblanco on some kind of convention.